README.md

```diff
@@ -1,3 +1,34 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+Built with Ruby 2.7.2, CSV, Rspec 3.12.0
+
+
```

Comment:
Could we have headers for each section as well?

Reply:
I added headers for all sections.